x-pack/plugins/lens/public/editor_frame_service/editor_frame/workspace_panel/geo_field_workspace_panel.tsx

```diff
@@ -0,0 +1,78 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+import { EuiPageContentBody, EuiText } from '@elastic/eui';
+import { FormattedMessage } from '@kbn/i18n/react';
+import { i18n } from '@kbn/i18n';
+import {
+  UiActionsStart,
+  VISUALIZE_GEO_FIELD_TRIGGER,
+} from '../../../../../../../src/plugins/ui_actions/public';
+import { getVisualizeGeoFieldMessage } from '../../../utils';
+import { DragDrop } from '../../../drag_drop';
+import { GlobeIllustration } from '../../../assets/globe_illustration';
+import './geo_field_workspace_panel.scss';
+
+interface Props {
+  fieldType: string;
+  fieldName: string;
+  indexPatternId: string;
+  uiActions: UiActionsStart;
+}
+
+const dragDropIdentifier = {
+  id: 'lnsGeoFieldWorkspace',
+  humanData: {
+    label: i18n.translate('xpack.lens.geoFieldWorkspace.workspaceLabel', {
```

Comment:
This is used to construct the screen reader message if the user picks up the field using keyboard and moves it to the drop zone:
<img width="896" alt="Screenshot 2021-05-06 at 15 51 38" src="https://user-images.githubusercontent.com/1508364/117309753-030c0400-ae83-11eb-8e60-cd26fbdb4309.png">

We can either add a special case for this here (extending `humanData` for a special message): https://github.com/elastic/kibana/blob/755c186c28cc3e70266d33bbf04fe6b90b86d872/x-pack/plugins/lens/public/drag_drop/providers/announcements.tsx#L530

or think about something that fits into the existing template and makes clear this will move the user to Maps.